src/status_im2/contexts/chat/messages/content/reactions/view.cljs

```diff
@@ -5,31 +5,79 @@
+        _show-sheet                               (when (and reaction-authors-list
+                                                             @show-reaction-authors-sheet?)
+                                                    (rf/dispatch [:dismiss-keyboard])
+                                                    (rf/dispatch
+                                                     [:show-bottom-sheet
+                                                      {:on-close
+                                                       (fn []
+                                                         (reset! show-reaction-authors-sheet? false)
+                                                         (rf/dispatch
+                                                          [:chat/clear-emoji-reaction-author-details]))
+                                                       :content
+                                                       (fn [] [drawers/reaction-authors
+                                                               reaction-authors-list
+                                                               reaction-authors-list-selected-reaction])
+                                                       :selected-item (fn []
+                                                                        user-message-content)
+                                                       :padding-bottom-override 0}]))]
+    [:<>
+     (when (seq reactions)
+       [rn/view
+        {:style {:margin-left    52
+                 :margin-bottom  12
+                 :flex-direction :row}}
+        (for [{:keys [own emoji-id quantity emoji-reaction-id]
+               :as   emoji-reaction} reactions]
+          ^{:key (str emoji-reaction)}
+          [rn/view {:style {:margin-right 6}}
+           [quo/reaction
+            {:emoji               (get constants/reactions emoji-id)
+             :neutral?            own
+             :clicks              quantity
+             :on-press            #(on-press own message-id emoji-id emoji-reaction-id)
+             :on-long-press       #(on-long-press chat-id
+                                                  message-id
+                                                  emoji-id
+                                                  show-reaction-authors-sheet?)
+             :accessibility-label (str "emoji-reaction-" emoji-id)}]])
+        [quo/add-reaction
+         {:on-press (fn []
+                      (rf/dispatch [:dismiss-keyboard])
+                      (rf/dispatch
+                       [:show-bottom-sheet
+                        {:content       (fn [] [drawers/reactions
+                                                {:chat-id    chat-id
+
+                                                 :message-id message-id}])
+                         :selected-item (fn []
+                                          user-message-content)}]))}]])]))
```

Comment:
:selected-item user-message-content

Reply:
It gets used as hiccup, It's expected to be a function.
If I removed the anonymous function, I get invalid arity: 0 error.